example/src/main/java/com/vinted/coper/example/PermissionExampleInBusinessSideViewModel.kt

```diff
@@ -19,7 +20,7 @@ class PermissionExampleInBusinessSideViewModel(
     val permissionRequestResultEvent: LiveData<PermissionResult> = _permissionRequestResultEvent
 
     fun onOnePermissionClicked(permission: String) {
-        launch {
+        launch(Dispatchers.IO) {
```

Comment:
Why do you need IO here? Are you blocking something?

Reply:
Its just for showing that it can